### ISSUE TITLE
Mask RCNN: Parameter Finetuning

### ISSUE THREAD

**Issue by yc0619** [open-mmlab/mmdetection#4799]
Hi,

My training data: 695 (original) -> 512 (resize) average 30-40 Objects, which in middle size and small size
My val data: 695(original) no resize, average 30-40 Objects

I have made nms_pre and max_per_img at train or test cfg little bigger.

I would like to ask, which parameter in Mask RCNN could I finetuning to adjust my quest? Whats your suggestion? Thanks!

**Comment by ZwwWayne** [comment]
You may change the img_scale of the resize parameter, anchor size, and the pyramid number of FPN. Or you can simply scale your image to 1333x800 and the performance should not be bad.

**Comment by yc0619** [comment]
jo, thanks!

**Comment by yc0619** [comment]
Still want to ask, if it is really need to do so, to shape the training image to 1333*800, cause my input is a square image, which length equals to width.

**Comment by yc0619** [comment]
> You may change the img_scale of the resize parameter, anchor size, and the pyramid number of FPN. Or you can simply scale your image to 1333x800 and the performance should not be bad.

Still want to ask, if it is really need to do so, to shape the training image to 1333*800, cause my input is a square image, which length equals to width.
